render/wgpu/src/blend.rs

```diff
@@ -4,8 +4,6 @@ use swf::BlendMode;
 
 #[derive(Enum, Debug, Copy, Clone)]
 pub enum ComplexBlend {
-    Multiply,   // Can't be trivial, 0 alpha is special case
```

Comment:
Were these comments incorrect?

Reply:
Kiiinda. 0 isn't specialcase, "not written yet" was. By making the default pixel color to white, it handles this case